pkg/doc/jose/jwe_test.go

```diff
@@ -105,7 +134,7 @@ func TestJSONWebEncryption_Serialize(t *testing.T) {
 					IV:  "TestIV",
 					Tag: "TestTag",
 					KID: "TestKID",
-					SPK: "TestSPK",
+					SPK: []byte(exampleSPK),
```

Comment:
Please use `EPK` instead of `SPK` which is not JWE/JWS compliant. `EPK` is compliant.

Reply:
Done